docs/ARMInstallationManagedIdentity.md

```diff
@@ -0,0 +1,59 @@
+# Installation via ARM Template for use with Azure API for FHIR and Azure Active Directory
+This article details provisioning and installation of the IoMT FHIR Connector for Azure to connect to Azure API for FHIR with a managed identity in the same subscription using an ARM template.
+
+## ARM Template Provisioning
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2Fiomt-fhir%2Fmaster%2Fdeploy%2Ftemplates%2Fmanaged-identity-azuredeploy.json" target="_blank">
+    <img src="https://azuredeploy.net/deploybutton.png"/>
+</a>
+
+An [ARM Template](../deploy/templates/managed-identity-azuredeploy.json) is provided for easy provisioning of an environment within Azure with the Azure API for FHIR. When executed, the ARM template will provision the following:
+
+* App Service Plan - The service plan for used for hosting the Azure Functions Web app.
+* Azure Web App - The web app running the Azure Functions responsible for normalization and FHIR conversion.
+* Azure Event Hubs - Two Event Hubs are deployed. One is the initial ingestion point for device data. The second receives normalized device data for further processing.
+* Azure Stream Analytics - Used to group and buffer the normalized data stream. Controls the end to end latency between device data ingested and landing the data in the configured FHIR server.
+* Azure Key Vault - Used for secret storage.  Event Hub Shared Access Keys and the OAuth client credentials are stored here.
+* Azure Storage - Used by the Azure Functions to track Event Hub processing watermark and also hosts the configuration files for device normalization mapping and FHIR conversion mapping.
+* App Insights - Used to record telemetry.
+* Managed Identity - an Azure Active Directory service identity for the IoMT FHIR Connector for Azure to use to connect to the Azure API for FHIR
+
+### Prerequisites
+To run this ARM template the following additional items must be set up before execution:
+
+* FHIR Server - An Azure API for FHIR instance using FHIR version R4
+
+### Parameters
+The following parameters are provided by the ARM template:
```

Comment:
There are two sections of parameters in the template: BASICS and SETTINGS. Below parameters are covering all parameters for SETTINGS section, but parameters for BASICS section are missing. Won't we need to document those?

Reply:
The BASICS aren't part of our template or under our control. Those are things that are added by Azure Portal as part of the custom template deployment screen.